tools/issue-labeler/src/IssueLabelerService/AzureSdkIssueLabelerService.cs

```diff
@@ -24,15 +24,15 @@ public class AzureSdkIssueLabelerService
         private static readonly ActionResult EmptyResult = new JsonResult(new IssueOutput { Labels = [], Answer = null, AnswerType = null });
         private readonly ILogger<AzureSdkIssueLabelerService> _logger;
         private readonly IConfiguration _config;
-        private readonly ITriageRAG _issueLabeler;
+        private readonly TriageRag _issueLabeler;
```

Comment:
Is this name accurate?   I'd assume not so much if we're splitting out labeling and answers.